attributes/geo:proj/schema.json

```diff
@@ -0,0 +1,173 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://zarr-specs.readthedocs.io/en/latest/extensions/attributes/projection/v1.0.0/schema.json",
+  "title": "Zarr Projection Attribute Extension",
+  "description": "Projection attribute extension for Zarr arrays and groups",
+  "type": "object",
+  "definitions": {
+    "projectionMetadataV1_0": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": ["string", "null"],
+          "description": "Authority:code identifier (e.g., EPSG:4326)",
+          "pattern": "^[A-Z]+:[0-9]+$"
+        },
+        "wkt2": {
+          "type": ["string", "null"],
+          "description": "WKT2 (ISO 19162) CRS representation"
+        },
+        "projjson": {
+          "oneOf": [
+            {
+              "$ref": "https://proj.org/schemas/v0.7/projjson.schema.json"
+            },
+            {
+              "type": "null"
+            }
+          ],
+          "description": "PROJJSON CRS representation"
+        },
+        "bbox": {
+          "type": "array",
+          "oneOf": [
+            {
+              "minItems": 4,
+              "maxItems": 4
+            },
+            {
+              "minItems": 6,
+              "maxItems": 6
+            }
```

Comment:
Are you suggesting that the bounding box could contain a third dimension? If so, that would increase the scope of this extension a lot because it would be necessary to define a convention for identifying the third dimension.

Reply:
Nope, this is a relicate from STAC extension. We remove the third dimension here.